tests/installation/await_install.pm

```diff
@@ -97,7 +97,16 @@ sub run {
             }
         }
         else {
-            assert_screen \@tags, $timeout;
+            # need end test if find screen not change for 300s
+            while (1) {
+                last if check_screen(\@tags, 20);
```

Comment:
`last` exits from `while(1)` loop - from comment I guess you want to exit from parent `while ($keep_trying)`
Didn't you want `$keep_trying = 0`?

Reply:
No need #keep_trying =0, just jump out of while(1) is enough.
The purpose of this change is trigger case failed quickly if something wrong, such as screen not change for long time. 
Normally the logic will hit line 107 (assert_screen \@tags, 10;) in case the screen freeze, finally throw exceptions by assert.